src/styles/typography.scss

```diff
@@ -1,5 +1,28 @@
+.lbh-body {
+  @include govuk-media-query($until: tablet) {
+    margin-top: 1em;
+  }
+}
+
 .lbh-heading-h1 {
   @include govuk-media-query($until: tablet) {
     font-size: 20px;
   }
 }
+
+.lbh-button {
+  @include govuk-media-query($until: tablet) {
+    width: 226px;
+    height: 51px;
+    font-size: 19px;
+    font-weight: 400;
+    line-height: 0.875rem;
+    margin-top: 1em;
+  }
+}
+
+.lbh-button__start-icon {
+  @include govuk-media-query($until: tablet) {
+    display: none;
+  }
+}
```

Comment:
Think it's best to keep this file just for any general font-specific styling we want like font size and weight which applies broadly, otherwise it's a bit mixed with spacing and display rules.